sdk/examples/wallet/update_alias_output.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will update the state metadata of an alias output.
+//! Rename `.env.example` to `.env` first.
+//!
+//! `cargo run --example update_alias_output --release`
+
+use std::{env, str::FromStr};
+
+use iota_sdk::{
+    types::block::output::{AliasId, AliasOutputBuilder, Output},
+    wallet::{account::FilterOptions, Result, Wallet},
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    //  This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    // Create the wallet
+    let wallet = Wallet::builder().finish().await?;
+
+    // Get the account we generated with `01_create_wallet`
+    let account = wallet.get_account("Alice").await?;
+
+    account.sync(None).await?;
+
+    // Set the stronghold password
+    wallet
+        .set_stronghold_password(&env::var("STRONGHOLD_PASSWORD").unwrap())
+        .await?;
+
+    // Replace with an AliasId
+    let alias_id = AliasId::from_str("0xc94fc4d280d63c7de09c8cc49ecefba6192e104d200ab7472db9e943e0feef7c")?;
+
+    // Get the alias output by it's alias id
+    let alias_output = account
+        .unspent_outputs(Some(FilterOptions {
+            output_types: Some(vec![4]),
```

Comment:
For the sake of the example, I think I would use AliasOutput::KIND or whatever it is, instead of a magic number